spec/test-outputs/www-integration.out.vcl

```diff
@@ -426,6 +459,43 @@ sub vcl_deliver {
   }
   # End dynamic section
 
+if (req.http.Cookie ~ "cookies_policy") {
+  if (req.http.Cookie:cookies_policy ~ "%22usage%22:true" && req.http.User-Agent !~ "^GOV\.UK Crawler Worker"){
+    if (table.lookup(active_ab_tests, "StartABusinessSegment") == "true") {
+      declare local var.variant STRING;
+      set var.variant = req.http.GOVUK-ABTest-StartABusinessSegment;
+
+      declare local var.sab_expiry TIME;
+      if (var.variant == "C") {
+        # If they have opted out of the banner or have visited the SaB checker
+        # Expire in 26 weeks (approx 6 months)
+        set var.sab_expiry = time.add(now, 26w);
+      } else {
+        # Otherwise, expire in half an hour, approx this session
+        set var.sab_expiry = time.add(now, 30m);
+      }
```

Comment:
Looks like an incomplete duplicate of lines 478-490 below (though with a good refactoring!).

Reply:
whoops :facepalm: